src/kontrol/kompile.py

```diff
@@ -132,7 +132,7 @@ def update_kompilation_digest() -> None:
         digest_dict = _read_digest_file(foundry.digest_file)
         digest_dict['kompilation'] = kompilation_digest()
         digest_dict['kontrol'] = VERSION
-        digest_dict['build-options'] = options.to_string()
+        digest_dict['build-options'] = options
```

Comment:
Are you sure there's an implicit `str` when dumping the JSON?

Reply:
I just tested, I need to wrap it in `str()`.